web_notify/readme/USAGE.rst

```diff
@@ -37,26 +37,28 @@ The notifications can bring interactivity with some buttons.
 
 The reload button is activated when sending the notification with:
 
-
-The action can be used using the ``action`` keyword:
+The action can be used using the ``action`` keyword and we can choose which name to
+give to our button with the ``button_name`` key in the action context `params` key:
 
 .. code-block:: python
 
-    action = self.env["ir.actions.act_window"]._for_xml_id('sale.action_orders')
-    action.update({
-        'res_id': self.id,
-        'views': [(False, 'form')],
-    })
+   action = self.env["ir.actions.act_window"]._for_xml_id('sale.action_orders')
+   action.update({
+      'res_id': self.id,
+      'views': [(False, 'form')],
+   })
+   action["context"].setdefault("params", {})
+   action["context"]["params"]["button_name"] = "Sales"
    self.env.user.notify_info('My information message', action=action)
 
 
-.. figure:: static/description/notifications_screenshot.gif
+.. figure:: ../static/description/notifications_screenshot.gif
    :scale: 80 %
    :alt: Sample notifications
 
 You can test the behaviour of the notifications by installing this module in a demo database.
 Access the users form through Settings -> Users & Companies. You'll see a tab called "Test web notify", here you'll find two buttons that'll allow you test the module.
 
-.. figure:: static/description/test_notifications_demo.png
+.. figure:: ../static/description/test_notifications_demo.png
```

Comment:
Try to put the image on the `img` folder:

```suggestion
.. figure:: ../static/img/test_notifications_demo.png
```

and rebuild the README (forcing it).